### PR TITLE
Am longlv new feature mobile UI design xml

### PR DESCRIPTION
**Pull request by longlv81**

### CODE REVIEW

File: app/src/am_longlv/res/values/strings.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<resources>
+    <string name="app_name">Asian1</string>
+    <string name="none"> </string>
+    <!--Issue name-->
+    <string name="uiDesignXml">2.UI Design XML</string>
```
Comment (huyhn30) [inline app/src/am_longlv/res/values/strings.xml:5]: fix name

File: app/src/am_longlv/res/values/strings.xml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    <!--Issue 2 string-->
+    <string name="facebookLogin">Facebook Login</string>
```
Comment (huyhn30) [inline app/src/am_longlv/res/values/strings.xml:7]: read again android convention

File: app/src/am_longlv/res/values/colors.xml [hunk 3]
```diff
@@ -3,4 +3,10 @@
     <color name="colorPrimary">#3F51B5</color>
     <color name="colorPrimaryDark">#4CAF50</color>
     <color name="colorAccent">#03DAC5</color>
+    <color name="lightBlue">#00BCD4</color>
+    <color name="darkBlue">#3F51B5</color>
+    <color name="darkerBlue">#00093E</color>
+    <color name="yellow">#FFEB3B</color>
+    <color name="darkYellow">#FFC107</color>
+    <color name="lightGreen">#8BC34A</color>
```
Comment (huyhn30) [inline app/src/am_longlv/res/values/colors.xml:11]: fix again name

**Review by huyhn30 — CHANGES_REQUESTED** [review]
Check same bug again

File: app/src/am_longlv/res/values/colors.xml (hunk 3 above)
Comment (longlv81) [reply app/src/am_longlv/res/values/colors.xml:11]: should I change name "lightBlue" to "colorLightBlue" or "color_light_blue" ?

File: app/src/am_longlv/com/example/asian1/MainActivity.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        initView();
+        onListener();
+    }
+
+    private void initView() {
+        mBtnDesignXML = findViewById(R.id.btnDesignXML);
+    }
+
+    private void onListener() {
+        mBtnDesignXML.setOnClickListener(new View.OnClickListener() {
```
Comment (huyhn30) [inline app/src/am_longlv/com/example/asian1/MainActivity.java:29]: use lambda

File: app/src/am_longlv/com/example/asian1/MainActivity.java [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+            @Override
+            public void onClick(View view) {
```
Comment (huyhn30) [inline app/src/am_longlv/com/example/asian1/MainActivity.java:31]: same

**Review by huyhn30 — CHANGES_REQUESTED** [review]
check same debug and merge code

File: app/src/am_longlv/com/example/asian1/MainActivity.java [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+                Intent intent = new Intent(MainActivity.this, UIDesignActivity.class);
```
Comment (huyhn30) [inline app/src/am_longlv/com/example/asian1/MainActivity.java:32]: Intent intent = new Intent(this, UIDesignActivity.class);

File: app/src/am_longlv/com/example/asian1/issue2/UIDesignActivity.java [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+        initView();
+        onListener();
+    }
+
+    private void onListener() {
+        mBtnFacebookLogin.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                Intent intent = new Intent(UIDesignActivity.this, FacebookLoginActivity.class);
```
Comment (huyhn30) [inline app/src/am_longlv/com/example/asian1/issue2/UIDesignActivity.java:31]: Intent intent = new Intent(this, FacebookLoginActivity.class);

**Review by huyhn30 — CHANGES_REQUESTED** [review]

File: app/src/am_longlv/res/layout/activity_calculator.xml [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+    android:orientation="vertical">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="20dp"
+        android:orientation="horizontal">
+
+        <TextView
+            android:id="@+id/tvNumber1"
```
Comment (huyhn30) [inline app/src/am_longlv/res/layout/activity_calculator.xml:14]: rename

File: app/src/am_longlv/res/layout/activity_calculator.xml [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="20dp"
+            android:text="@string/number_one"
+            android:textColor="@android:color/black" />
+
+        <EditText
+            android:id="@+id/edtNumber1"
```
Comment (huyhn30) [inline app/src/am_longlv/res/layout/activity_calculator.xml:22]: rename

**Review by huyhn30 — CHANGES_REQUESTED** [review]
check all id name again

**Review by huyhn30 — APPROVED** [review]

**Pull request merged**